src/js/ShoppingCart.mjs

```diff
@@ -13,18 +13,17 @@ function cartItemTemplate(item) {
             <p class="cart-card__color">${item.Colors[0].ColorName}</p>
             <p class="cart-card__quantity">qty: 1</p>
             <p class="cart-card__price">$${item.FinalPrice}</p>
-    </li>`
-
+    </li>`;
 }
 
-export default class ShoppingCart{
-    constructor(key,parentSelector){
-        this.key = key;
-        this.parentSelector = parentSelector;
-    }
-    renderCartContents() {
-        const cartItems = getLocalStorage(this.key);
-        const htmlItems = cartItems.map((item)=> cartItemTemplate(item));
-        document.querySelector(this.parentSelector).innerHTML = htmlItems.join("");
-    }
+export default class ShoppingCart {
+  constructor(key, parentSelector) {
+    this.key = key;
+    this.parentSelector = parentSelector;
+  }
+  renderCartContents() {
+    const cartItems = getLocalStorage(this.key) || [];
```

Comment:
Again, make sure your previous fixes are applied to new stuff. The instructor solutions _do not_ contain bug fixes.